### PR TITLE
added dotknot package

### PR DESCRIPTION
**Pull request by HassanAmr**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/dotknot/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  run:
+    - python
+    - viennarna ==1.8.5
+
+test:
+  source_files:
+    - sample.fasta
+
+  commands:
+    - dotknot.py sample.fasta
```
Comment (bgruening) [inline recipes/dotknot/meta.yaml]: can you simply create a sample.fasta file on the fly with `echo '>foo\nacgtcgttgtgt' > foo.fasta && dotknot.py foo.fasta`?

Comment (bgruening) [inline recipes/dotknot/meta.yaml]: similar to this, not tested ;)

Comment (HassanAmr) [inline recipes/dotknot/meta.yaml]: Sure, 1 minute

**Comment by jerowe** [comment]
@HassanAmr , should we keep this open?

**Comment by HassanAmr** [comment]
@jerowe, yes please. Unless you want me to close it until I figure out how to fix the problem locally first.

**Comment by johanneskoester** [comment]
We  have made changes to our build system (migrated from Travis CI to  Circle CI). If you would like to continue working on this PR, please  reopen this PR and update with latest upstream/master.

**Pull request closed without merging**